docs/source/development-guide/schema-deployment/schema-changes.rst

```diff
@@ -0,0 +1,17 @@
+Schema Change Process
+============
+
+1. All database changes will be approved via pull request by Sean Goggins, following these basic principles:
+        - Avoid redundant data collection
+        - Ensure consistent naming
+        - Ensure logical and consistent breakdown of tables versus columns in a table
+        - Ensure foreign key and other rules are implemented
+        - Ensure the four pieces of Augur metadata are added
+
+2. Production Deployment involves the following steps
```

Comment:
This would probably be better served as part of a larger "deployment process" document.